src/docs/perf/app-size.md

```diff
@@ -124,6 +108,70 @@ IPAs are commonly larger than APKs as explained
 in [How big is the Flutter engine?][], a
 section in the Flutter [FAQ][].
 
+## Breaking down the size
+
+Starting in Flutter version 1.22 and in DevTools starting in version 0.9.1,
+a size analysis tool is included to help developers understand the breakdown
+of the release build of their application.
+
+{{site.alert.warning}}
+  As stated in the [checking total size](#checking-the-total-size) section
+  above, an upload package is not representative of your end users' download
+  size. Be aware that redundant native library architectures and asset densities
+  seen in the breakdown tool can be filtered by the Play Store and App Store.
+{{site.alert.end}}
+
+The size analysis tool is invoked by passing the `--analyze-size` flag when
+building:
+
+- `flutter build apk --analyze-size`
+- `flutter build appbundle --analyze-size`
+- `flutter build ios --analyze-size`
+- `flutter build linux --analyze-size`
+- `flutter build macos --analyze-size`
+- `flutter build windows --analyze-size`
+
+This build is different from a standard release build in 2 ways.
+
+1, the tool will compile Dart in a way that records code size usage of Dart
+packages. 2, the tool will display a high level summary of the size breakdown
+in the terminal and leave a `*-code-size-analysis_*.json` file for more detailed
+analysis in DevTools.
+
+In addition to analyzinge a single build, 2 builds can also be diffed by
+loading 2 `*-code-size-analysis_*.json` files into DevTools. See
+[DevTools documentation][] for details.
+
+{% include app-figure.md image="perf/size-summary.png" alt="Size summary of an Android application in terminal" %}
+
+Through the summary, you can get a quick idea of the size usage per category
+(such as asset, native code, Flutter libraries etc). The compiled Dart
+native library is further broken down by package for quick analysis.
+
+{{site.alert.warning}}
+  This tool on iOS creates a .app rather than an IPA. It also contains bitcode
+  which will drastically increase the .framework file sizes in the .app.
+
+  Use this tool to evaluate the relative size of the .app's content. To get
+  a closer estimate of the download size, reference the
+  [Estimating total size](#estimating-total-size) section above.
+{{site.alert.end}}
+
+### Deeper analysis in DevTools
+
+The `*-code-size-analysis_*.json` file produced above could be further
+analysised in deeper detail in DevTools where a tree or a treemap view could
```

Comment:
r/analysised/analyzed

Reply:
doh! added a spell check plugin to vscode 😂